src/map/lapack2flash/FLASH_trsyl.c

```diff
@@ -71,8 +72,6 @@
   if ( *sgn == 1 ) sgn_fla = FLA_ONE;                           \
   else             sgn_fla = FLA_MINUS_ONE;                     \
                                                                 \
-  blocksize = min( FLASH_get_preferred_blocksize(), *ldim_A );  \
-                                                                \
   FLA_Bool toggle = FLASH_Check_offload_to_gpu( blocksize, *m,  \
                           *n, FLASH_get_tile_offload() );       \
```

Comment:
I noticed this is currently based on m and n. But A is made from m and m. Not sure if I should change this to reflect

Reply:
I think this is OK - unless you don't?